backend/apps/github/api/internal/nodes/user.py

```diff
@@ -27,6 +28,19 @@
 class UserNode:
     """GitHub user node."""
 
+    @strawberry.field
+    def badges(self) -> list[BadgeNode]:
+        """Return user badges."""
+        user_badges = (
+            self.badges.select_related("badge")
```

Comment:
The relation doesn't look right, you ask for `self.badges` and then still need to access `.badge` attribute after that.